### PR TITLE
Bottom the footer

### PR DESCRIPTION
**Pull request by ysabri**
@kreloff-01 You can top the footer now

### CODE REVIEW

File: src/components/Header/HeaderBar.js [hunk 1]
```diff
@@ -20,15 +20,15 @@ class HeaderBar extends React.Component {
...
+      cookies: new Cookies(),
+      loggedIn: '',
+      currUser: '',
+      refLoaded: false,
+      checked: []
+    };
 
     this.setCookies = this.setCookies.bind(this)
-    this.setRefLoaded= this.setRefLoaded.bind(this)
+    this.setRefLoaded = this.setRefLoaded.bind(this)
```
Comment (kreloff-01) [inline src/components/Header/HeaderBar.js:31]: nice catch here

File: src/components/Header/HeaderBar.js [hunk 2]
```diff
@@ -70,33 +70,42 @@ class HeaderBar extends React.Component {
...
         <div><NavBarComponent loggedIn={this.state.loggedIn}/></div>
-        <Switch>
-          <ProtectedRoute exact path='/' loggedIn={ this.state.loggedIn } currUser={ this.state.currUser } component={HomePage} />
-          <Route path="/signup" render={(props) => <SignUpParentForm {...props} cookies={ this.state.cookies } setCookies={this.setCookies} />} />
-          <Route path="/signupCongrats" render={(props) => <SignupCongrats {...props} cookies={ this.state.cookies } setCookies={this.setCookies}/>} />
-          <Route path="/dashboard" component={Dashboard} user={ this.state.user } />
-          <Route path="/messages/:convoId" render={() => <Messenger/>}/>
-          <Route path="/commission/:id" render={(props) => <Commission {...props} currUser={this.state.loggedIn} cookies={ this.state.cookies } setCookies={this.setCookies} refLoaded={this.state.refLoaded} setRef={this.setRefLoaded}/>} />
-        </Switch>
+        <div className="content-container">
```
Comment (kreloff-01) [inline src/components/Header/HeaderBar.js:92]: I  assume this css just was  useless then?

**Comment by kreloff-01** [comment]
lgtm

File: src/components/Header/HeaderBar.js (hunk 1 above)
Comment (ysabri) [reply src/components/Header/HeaderBar.js:31]: hahaha I have an auto formatter in my IDE

File: src/components/Header/HeaderBar.js (hunk 2 above)
Comment (ysabri) [reply src/components/Header/HeaderBar.js:92]: nah, I created it for the dashboard, I just moved it up so it applies to all pages now

**Pull request merged**